_episodes/08-defensive.md

```diff
@@ -531,11 +531,13 @@ This violates another important rule of programming:
 > >
 > > def range_overlap(ranges):
 > >     '''Return common overlap among a set of [low, high] ranges.'''
+> >     if not ranges:
```

Comment:
What do you think about `if len(ranges) == 0` instead? I know experienced Python users like the simplicity of this, but to a novice it feels more clear that an empty list has length 0.